software/include/enso/pipe.h

```diff
@@ -124,11 +125,35 @@ class Device {
    * @brief Allocates a TX pipe.
    *
    * @param buf Buffer address to use for the pipe. It must be a pinned
-   *            hugepage. If not specified, the buffer is allocated internally.
-   * @return A pointer to the pipe. May be null if the pipe cannot be created.
+   *            hugepage. If not specified, the buffer is allocated
+   *            internally.
+   * @return A pointer to the pipe. May be null if the pipe cannot be
+   *         created.
    */
   TxPipe* AllocateTxPipe(uint8_t* buf = nullptr) noexcept;
 
+  /**
+   * @brief Retrieves the number of fallback queues for this device.
+   */
+  int GetNbFallbackQueues() noexcept;
+
+  /**
+   * @brief Sets the round robin status for the device.
+   *
+   * @param round_robin Whether to enable or disable round robin.
+   *
+   * @return 0 on success, -1 on failure.
+   */
+  int SetRrStatus(bool round_robin) noexcept;
+
+  /**
+   * @brief Gets the round robin status for the device.
+   *
+   * @return 0 if round robin is disabled, 1 if round robin is enabled, -1 on
+   *         failure.
+   */
+  bool GetRrStatus() noexcept;
+
```

Comment:
Can you move these functions closer to the similar ones (after  `DisableRoundRobin`)?